File: RepoTransBench/target_projects/Python/Java/jtsang4_claude-to-chatgpt/src/test/java/com/example/original/ModelsTest.java
package com.example.original;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
import java.util.*;

public class ModelsTest {

    @Test
    public void testModelsListExists() {
        List<String> modelsList = ModelsMockData.modelsList;
        assertNotNull(modelsList);
        assertTrue(modelsList instanceof List);
    }

    @Test
    public void testModelMapExists() {
        Map<String,String> modelMap = ModelsMockData.modelMap;
        assertNotNull(modelMap);
        assertTrue(modelMap instanceof Map);
    }
}

class ModelsMockData {
    public static List<String> modelsList = Arrays.asList("model1","model2","model3");
    public static Map<String,String> modelMap = Map.of(
        "gpt-3.5-turbo-0613", "claude-2", 
        "gpt-4-0314", "claude-v1"
    );
}

File: RepoTransBench/target_projects/Python/Go/jtsang4_claude-to-chatgpt/tests/original/models_test.go
package original

import (
	"testing"
	"github.com/stretchr/testify/assert"
	"reflect"
)

func TestModelsListExists(t *testing.T) {
	// Simulate: models_list is a []string
	modelsList := []string{"model1", "model2"}
	assert.Equal(t, reflect.TypeOf(modelsList).Kind(), reflect.Slice)
}

func TestModelMapExists(t *testing.T) {
	// Simulate: model_map is a map[string]string
	modelMap := map[string]string{"gpt-3.5-turbo-0613": "claude-2"}
	assert.Equal(t, reflect.TypeOf(modelMap).Kind(), reflect.Map)
}